app/app.ts

```diff
@@ -28,4 +30,12 @@ app.use('/favicon.ico', (_req, res) => {
 });
 app.use(staticFilesPath, express.static(path.join(__dirname, 'dist')));
 
+async function readIdpList(): Promise<string> {
+  const idplistDefaultPath = './config/idplist-default.json';
+  const idplistOverridePath = './config/idplist-override.json';
```

Comment:
would anyone prefer if overrides were in `idplist-local.json`

Reply:
Could change it to be named local instead - would match better with existing config library patterns. 👍